website/blog/2019-07-15-cucumberjs-v5-released.md

```diff
@@ -0,0 +1,108 @@
+---
+title: WebdriverIO V5 now also supports CucumberJS
+author: Wim Selles
+authorURL: http://github.com/wswebcreation
+authorImageURL: https://avatars2.githubusercontent.com/u/11979740?s=460&v=4
+---
+
+We are pleased to announce that we now have support for CucumberJS in WebdriverIO V5!!!! 
+This has been a great challenge for the projectcommitters so we're all very thankful for the time and effort they put into this.
+So normally we would say
+
+    npm install @wdio/cucumber-framework --save-dev
+    
+and go with the flow, but this time it is different. When you upgrade to the latest version of the Cucumber framework you also need to upgrade to the **latest version of WebdriverIO**. 
+In this blog post we want to give you some guidelines on how to do that.
+
+## Where to start
+Upgrading to the latest version of the Cucumber framework isn't that simple, because as said, you also need to migrate to version 5 of WebdriverIO.
+To understand what you need to do we have created some steps you might want to follow which will make the migration a lot easier for you. 
+But before we explain the steps, you first need to understand the differences between WebdriverIO V4 and WebdriverIO V5
+
+### W3C support
+WebdriverIO is now fully supporting the W3C protocol, this has a lot of advantages, but for your existing scripts some downsides. 
+One of the downsides might be that you are using methods that are based on the JSONWire Protocol, that are not supported by the newest drivers like for example ChromeDriver 74+.
+This might result in errors like for example `browser.positionClick() is not a function`. If you see this error you are using a not supported method for the W3C supported Driver. 
+See the API documentation [here](https://webdriver.io/docs/api.html) to see which command is a Webdriver Protocol (W3C) or a JSONWire protocol command.
```

Comment:
Let's also mention that all `browser` and `element` commands (https://webdriver.io/docs/api.html) are build so that they work for both protocols. Nothing changes here for the user.

Reply:
I changed it, thanks